### ISSUE TITLE
PR reminders fail when reviewer is not in github-provider-map

### ISSUE THREAD

**Issue by zdc217** [DavideViolante/pr-reviews-reminder-action#98]
Using Teams, The PR reminder code fails if the user cannot be found in the mapping file, due to omission or case sensitivity.

We were able to determine the cause of the failure by capturing packets with Wireshark and comparing working and failing payloads. 

The problems is the user is still added to the "entities" block in the AdaptiveCard but without the `<id></id>` element. This results in a 200 response with a message body that contains a 400 error from Teams, which we confirmed with curl. The PR reminder code does not report an error (See #97).

We tested and created the examples below with a PR that had only tzturner as a reviewer. 

Inputs (note that tzturner is not present in the provider map):
```yml
  pr-reviews-reminder:
    runs-on: ubuntu-latest
    steps:
    - uses: DavideViolante/pr-reviews-reminder-action@v2.5.0
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      with:
        webhook-url: ${{ secrets.TEAMS_WEBHOOK }}
        provider: 'msteams'
        github-provider-map: >-
          foo:foo@illinois.edu,
          bar:bar@illinois.edu
```

Failing Adaptive card json captured using Wireshark from pr-reviews-reminder-action. Note the missing `<id></id>` element in the "entities" block which causes Teams to report the following error: `Webhook message delivery failed with error: Microsoft Teams endpoint returned HTTP error 400 with ContextId MS-CV=42RkMnqSMESvU3jecxUBgA.0..'`
```json
{
    "type": "message",
    "attachments": [
        {
            "contentType": "application/vnd.microsoft.card.adaptive",
            "content": {
                "type": "AdaptiveCard",
                "body": [
                    {
                        "type": "TextBlock",
                        "text": "Hey @tzturner, the PR \"Test reminders\" is waiting for your review: [https://github.com/techservicesillinois/secdev-template-repository/pull/5](https://github.com/techservicesillinois/secdev-template-repository/pull/5)  \n",
                        "wrap": true
                    }
                ],
                "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
                "version": "1.0",
                "msteams": {
                    "width": "Full",
                    "entities": [
                        {
                            "type": "mention",
                            "text": "<at>tzturner</at>",
                            "mentioned": {
                                "name": "tzturner"
                            }
                        }
                    ]
                }
            }
        }
    ]
}
```
The correct AdaptiveCard json should have an empty entities block in this case. The following AdaptiveCard json works and would resolve the issue we had:
```json
{
    "type": "message",
    "attachments": [
        {
            "contentType": "application/vnd.microsoft.card.adaptive",
            "content": {
                "type": "AdaptiveCard",
                "body": [
                    {
                        "type": "TextBlock",
                        "text": "Hey @tzturner, the PR \"Test reminders\" is waiting for your review: [https://github.com/techservicesillinois/secdev-template-repository/pull/5](https://github.com/techservicesillinois/secdev-template-repository/pull/5)  \n",
                        "wrap": true
                    }
                ],
                "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
                "version": "1.0",
                "msteams": {
                    "width": "Full",
                    "entities": []
                }
            }
        }
    ]
}
```
Can we please omit PR reviewers from the entities block when they are not found in the provider map?

Thanks,
The University of Illinois Cybersecurity Development Team

**Comment by edthedev** [comment]
On further discussion - It would be great to add a warning to the log output when a requested reviewer is not in the provider map.

**Comment by DavideViolante** [comment]
Thank you for reporting!
This should now be fixed, please try the v2.6.0 and let me know.
Also #97 is fixed.

**Comment by edthedev** [comment]
We've updated to 2.6.0 and all is working well. Thanks @DavideViolante !
